### PR TITLE
Improve SWA docs

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

Part of https://github.com/PyTorchLightning/pytorch-lightning/issues/8654

### Does your PR introduce any breaking changes? If yes, please list them.

None

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [n/a] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [n/a] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]

**Comment by chanshing** [comment]
@carmocca Thank you for improving the doc!

Could you also clarify what are the defaults when using `Trainer(stochastic_weight_avg=True)`? Are they those of `StochasticWeightAveraging`? And in that case, what is the default value for `swa_lrs`? 

If I understood the code correctly, it seems to be the last learning rate(s) used -- is this correct?
https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/callbacks/stochastic_weight_avg.py#L162-L189

**Comment by carmocca** [comment]
> Could you also clarify what are the defaults when using `Trainer(stochastic_weight_avg=True)`? Are they those of `StochasticWeightAveraging`?

Yes

> If I understood the code correctly, it seems to be the last learning rate(s) used -- is this correct?

The current learning rate for the optimizer at the time the SWA procedure starts.

I'll update the docs with these

**Review by SeanNaren — APPROVED** [review]
